testsuite/features/secondary/min_salt_migration.feature

```diff
@@ -0,0 +1,174 @@
+    Then I should see a "pending" text
+    And I accept "salt-sles-migration-test" key
+    Then I should not see a "pending" text
+    And I wait until onboarding is completed for "salt-sles-migration-test"
+
+  Scenario: Create the bootstrap repository for the SLES 15 KVM VM
+    When I create the bootstrap repository for "sle_minion" on the server
+
+  Scenario: SLES 15 KVM VM hardware refresh
+    Given I am on the Systems overview page of this "salt-sles-migration-test"
+    When I follow "Hardware"
+    And I click on "Schedule Hardware Refresh"
+    Then I should see a "You have successfully scheduled a hardware profile refresh" text
+    And I wait until event "Hardware List Refresh scheduled by admin" is completed
+
+@susemanger
+  Scenario: Subscribe the SLES 15 KVM VM to the SLES 15 SP4 channel and enable client tools
+    Given I am on the Systems overview page of this "salt-sles-migration-test"
+    When I follow "Software" in the content area
+    And I follow "Software Channels" in the content area
+    And I wait until I do not see "Loading..." text
+    And I check radio button "SLE-Product-SLES15-SP4-Pool for x86_64"
+    And I wait until I see "SLE-Module-Basesystem15-SP4-Pool for x86_64" text
+    And I check "SLE-Manager-Tools15-Pool for x86_64 SP4"
+    And I check "SLE-Manager-Tools15-Updates for x86_64 SP4"
+    And I click on "Next"
+    Then I should see a "Confirm Software Channel Change" text
+    When I click on "Confirm"
+    Then I should see a "Changing the channels has been scheduled." text
+    When I follow "scheduled" in the content area
+    And I wait until I see "1 system successfully completed this action." text, refreshing the page
+
+@uyuni
+  Scenario: Subscribe the SLES 15 KVM VM to the SLES 15 SP4 channel and enable client tools
+    Given I am on the Systems overview page of this "salt-sles-migration-test"
+    When I follow "Software" in the content area
+    And I follow "Software Channels" in the content area
+    And I wait until I do not see "Loading..." text
+    And I check radio button "SLE-Product-SLES15-SP4-Pool for x86_64"
+    And I wait until I see "SLE-Module-Basesystem15-SP4-Pool for x86_64" text
+    And I check "Uyuni Client Tools for SLES15 SP4 x86_64"
+    And I click on "Next"
+    Then I should see a "Confirm Software Channel Change" text
+    When I click on "Confirm"
+    Then I should see a "Changing the channels has been scheduled." text
+    When I follow "scheduled" in the content area
+    And I wait until I see "1 system successfully completed this action." text, refreshing the page
+
+  Scenario: Do some basic testing on the SLES 15 KVM VM without Salt bundle
+    Given I am on the Systems page
+    When I follow "salt-sles-migration-test.tf.local"
+    And I follow "Software" in the content area
+    And I follow "Install"
+    And I enter "venv-salt-minion" as "list_4021519721_filterval"
+    And I click on "list_4021519721_filtername"
```

Comment:
```suggestion
    And I enter "venv-salt-minion" as the filtered package name
    And I click on the filter button
```

Reply:
Nice. That was actually the step I was looking for but it was not mentioned in https://github.com/uyuni-project/uyuni/blob/master/testsuite/documentation/cucumber-steps.md. Maybe we can add it there.